lib/hq/graphql/object.rb

```diff
@@ -102,6 +106,15 @@ def association_required?(association)
             validation.attributes.any? { |a| a.to_s == association.name.to_s }
           end
         end
+
+        def null_from_field?(name)
+          field = get_field(name.camelize(:lower))
+          if field
+            field.instance_variable_get("@return_type_null")
```

Comment:
I'm not sure we need to access an internal instance variable to accomplish this

Reply:
I didn't find the accessor for the instance `@return_type_null` attribute in [Field](https://www.rubydoc.info/gems/graphql/GraphQL/Schema/Field). Is there another way to access it?

Reply:
I don't know what "return_type_null" is, or why it needs accessed. I thought that the null logic just needs fixed here so that it works with belongs to and has one associations https://github.com/OneHQ/hq-graphql/blob/master/lib/hq/graphql/object.rb#L100